### ISSUE TITLE
Sharing table

### ISSUE THREAD

**Issue by SkaeX** [BrainMaestro/FCE#173]
@BrainMaestro, I'm not a fan of sharing a table between the components. I'll change it at the slightest chance I get.

**Comment by tenorz007** [comment]
Sharing table with components?

**Comment by SkaeX** [comment]
There is a table that is being reused and I'm not a fan. ⚠️

**Comment by tenorz007** [comment]
Reused? What table!

**Comment by SkaeX** [comment]
We have a component called `BulmaTable` and it is being used to display `section`, `semesters`, and `users`.

**Comment by BrainMaestro** [comment]
Why are you not a fan. That's the whole point of components. Reuse. Would you prefer to create a different table for each page that requires data displayed in table form? That defeats the whole purpose then.

**Comment by SkaeX** [comment]
No, that's not my line of thought. I was thinking we should have `Section` `component` and a `SectionTable` `component`. Are you with me?

**Comment by SkaeX** [comment]
In summary, that table is confusing me. 😄

**Comment by BrainMaestro** [comment]
Then it should be made clearer?

**Comment by SkaeX** [comment]
Should be changed! I can't imagine having a `Detail` component just like the `Table` component.

**Comment by BrainMaestro** [comment]
So you want to have multiple tables then?

**Comment by SkaeX** [comment]
Maybe.

**Comment by BrainMaestro** [comment]
Okay then
